airbyte-integrations/connectors/source-plaid/source_plaid/schemas/balance.json

```diff
@@ -8,6 +8,10 @@
     "current": { "type": "number" },
     "iso_currency_code": { "type": ["string", "null"] },
     "limit": { "type": ["number", "null"] },
-    "unofficial_currency_code": { "type": ["string", "null"] }
+    "unofficial_currency_code": { "type": ["string", "null"] },
+    "name": { "type": ["string"] },
+    "official_name": { "type": ["string","null"] },
+    "plaid_item_id": { "type": ["string"] },
+    "plaid_institution_id": { "type": ["string","null"] }
```

Comment:
I'm wondering if it might be better to mimic the API response structure in the schema rather than denormalizing the object. This will lead to a balance schema such as: 
```json
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "type": "object",
  "required": ["account_id", "current"],
  "properties": {
    "account_id": { "type": "string" },
    "official_name": { "type": ["string","null"] },
    "plaid_item_id": { "type": ["string"] },
    "plaid_institution_id": { "type": ["string","null"] },
    "balances": {
      "type": "object",
      "required": ["current", "name"],     
      "properties": {
        "current": { "type": "number" },
        "name": { "type": ["string"] },
        "available": { "type": ["number", "null"] },
        "iso_currency_code": { "type": ["string", "null"] },
        "limit": { "type": ["number", "null"] },
        "unofficial_currency_code": { "type": ["string", "null"] }
      }
    }
  }
}
```